src/main/resources/org/forgerock/openam/auth/nodes/SetProfilePropertyNode.properties

```diff
@@ -18,8 +18,15 @@
 
 nodeDescription=Set Profile Property
 properties=Properties
-properties.help=The user profile properties to set. Key = profile attribute name, Value = shared State object name or \
-  "literal".
-transientProperties=Transient properties
-transientProperties.help=The user profile properties to set. Key = profile attribute name, Value = transient state \
-  object name.
+properties.help=The user profile properties to set. Key = profile property name, Value = shared State object name or "literal".
+transientProperties=Transient Properties
+transientProperties.help=The user profile properties to set. Key = profile property name, Value = shared State object\
```

Comment:
profile has attributes, session has properties.

Reply:
Understood, then maybe this node should be called Set Profile Attributes Node?